src/EditorFeatures/CSharpTest2/EmbeddedLanguages/RegularExpressions/CSharpRegexParserTests.cs

```diff
@@ -148,17 +140,6 @@ private void TryParseSubTrees(
             {
                 regex = new Regex(token.ValueText, options);
             }
-            catch (IndexOutOfRangeException) when (allowIndexOutOfRange)
-            {
-                // bug with .NET regex parser.  Can happen with patterns like: (?<-0
-                Assert.NotEmpty(tree.Diagnostics);
-                return treeAndText;
-            }
-            catch (NullReferenceException) when (allowNullReference)
-            {
-                // bug with .NET regex parser.  can happen with patterns like: (?(?S))
-                return treeAndText;
-            }
```

Comment:
workarounds for when we found regex strings that would crash .net :)   these have been fixed in .net core, so we don't need this.